src/tests/account/test_secp256r1.cairo

```diff
@@ -0,0 +1,141 @@
+use openzeppelin::account::utils::secp256r1::{
+    SyscallResultTrait, Secp256r1Point, DebugSecp256r1Point, Secp256r1PointSerde,
+    Secp256r1PointPartialEq, Secp256r1PointStorePacking as StorePacking, secp256r1_new_syscall
+};
+use starknet::secp256_trait::Secp256PointTrait;
+use starknet::secp256r1::Secp256r1Impl;
+
+#[test]
+fn test_pack_big_secp256r1_points() {
+    let (big_point_1, big_point_2) = get_points();
```

Comment:
Are these points big points? The idea of this test is to check that the packing is right even for big values, maybe it makes sense to use just big values even if they are not valid points.

Reply:
Can you define big points or point me in the right direction so that I can build values that would make sense for that test?

Reply:
A point with x big enough for the high part of the u256 to use as many bits as possible would be the best case, but making sure that at least the high part is not 0 should suffice.